<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmqxForm</class>
 <widget class="QWidget" name="EmqxForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 1px solid #C2C7CB;
    position: absolute;
    border-image: url(:/pic/箭头.png);
}
QTabWidget::tab-bar {
    alignment: left;  
	color:red;
} 
/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    border-bottom-color: #F16B8C; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
	margin:2px;
}
 
QTabBar::tab:selected,QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
	border-bottom:2px solid #F16B8C;
}
 
QTabBar::tab:selected {
	color:#8e354a;
    border-bottom-color: #F16B8C; /* same as pane color */
}</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/server.png</normaloff>:/server.png</iconset>
      </attribute>
      <attribute name="title">
       <string>EMQX管理</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWebView" name="webView">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="url">
            <url>
             <string>http://127.0.0.1/</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/cpu.png</normaloff>:/cpu.png</iconset>
      </attribute>
      <attribute name="title">
       <string>客户端</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="styleSheet">
          <string notr="true">QTableWidget{
	background:transparent;
	border-left:none;
	border-top:none;
}
QHeaderView::section
{
	background-color: transparent;
	border:none;
	/*margin:5px;*/
  	border-bottom:1px solid #91989f;
	border-top:1px solid #bdc0ba;
}
QTableWidget::item{
	border:none;
	border-bottom:1px solid #bdc0ba;
}
/*选中item*/
QTableWidget::item:selected{
color:#FFFFFF;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #86473F);
}
/*
悬浮item
QTableWidget::item:hover{
background:#5B5B5B;
}*/
</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>客户端ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
          <property name="background">
           <color alpha="249">
            <red>255</red>
            <green>170</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/cpu.png</normaloff>:/cpu.png</iconset>
          </property>
         </column>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>IP地址</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>当前订阅数量</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>连接状态</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>所在节点</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>接入时间</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>创建时间</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/zhuti.png</normaloff>:/zhuti.png</iconset>
      </attribute>
      <attribute name="title">
       <string>主题</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget2">
         <property name="styleSheet">
          <string notr="true">QTableWidget{
	background:transparent;
	border-left:none;
	border-top:none;
}
QHeaderView::section
{
	background-color: transparent;
	border:none;
	/*margin:5px;*/
  	border-bottom:1px solid #91989f;
	border-top:1px solid #bdc0ba;
}
QTableWidget::item{
	border:none;
	border-bottom:1px solid #bdc0ba;
}
/*选中item*/
QTableWidget::item:selected{
color:#FFFFFF;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #86473F);
}
/*
悬浮item
QTableWidget::item:hover{
background:#5B5B5B;
}*/
</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>主题</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>等级(qos)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>节点</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>订阅者ID</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/dingyue.png</normaloff>:/dingyue.png</iconset>
      </attribute>
      <attribute name="title">
       <string>订阅</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/debug.png</normaloff>:/debug.png</iconset>
      </attribute>
      <attribute name="title">
       <string>测试</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
